website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -26,12 +27,19 @@ that can either be a namespace or mount.
 - `path` `(string: "")` - Path of the mount or namespace to apply the quota.
   A blank path configures a global lease count quota. For example `namespace1/`
   adds a quota to a full namespace, `namespace1/auth/userpass` adds a quota to
-  `userpass` in `namespace1`. Updating this field on an existing quota can have
-  "moving" effects. For example, updating `auth/userpass` to
-  `namespace1/auth/userpass` moves this quota from being a global mount quota to a
-  namespace specific mount quota. Quotas on a non-root namespace are not inherited by child
+  `userpass` in `namespace1`, and `namespace1/kv-v2/data/foo/bar` adds a quota to
+  a specific secret on a K/V v2 mount in `namespace1`. A trailing glob (`*`) can also
+  be added as part of the path after the mount to match paths that share the same prefix
+  prior to the glob. `namespace1/kv-v2/data/foo/*` would match both `namespace1/kv-v2/data/foo/bar`
+  and `namespace1/kv-v2/data/foo/baz`. Updating this field on an existing
+  quota can have "moving" effects. For example, updating `namespace1` to
+  `namespace1/auth/userpass` moves this quota from being a namespace quota to a
+  namespace specific mount quota. Non-global quotas are not inherited by child
```

Comment:
Nice change to remain consistent with phrasing! The prior switch between global and root namespaces is a bit confusing.